Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3-slim
```

Comment:
Yeah, probably should be. Looks like Sugardough is partway through a Py2 to 3 transition, but I don't think we really want to jump on the Py3 train